.gitignore

```diff
@@ -13,7 +13,6 @@
 
 # misc
 .DS_Store
-.env
```

Comment:
remove .env file from gignore , so that later on you will not get error

Reply:
yes i have removed it.